berkeley-function-call-leaderboard/.env.example

```diff
@@ -0,0 +1,20 @@
+# [OPTIONAL] Required for LLM generation step
+# Provide the API key for the model(s) you intend to use
+OPENAI_API_KEY=sk-XXXXXX
+MISTRAL_API_KEY=
+FIREWORKS_API_KEY=
+ANTHROPIC_API_KEY=
+NVIDIA_API_KEY=nvapi-XXXXXX
+YI_API_KEY=
+
+COHERE_API_KEY=
+USE_COHERE_OPTIMIZATION=False # True/False
```

Comment:
Good comment here. In the check,
```python
if os.getenv("USE_COHERE_OPTIMIZATION") == "True":
```
it check if the string is "True" or not. But if the user mistakenly input "true", it will goes to unoptimized version, and current code is silent and doesn't error out and says "true" is an invalid input. 

So, nit on this and one way to do parameter validation is to have "if, elif, else" and raise exception pattern in the if-block.

Reply:
good point

Reply:
resolved.